bedrock/firefox/templates/firefox/australis/fx36/tour.html

```diff
@@ -2,26 +2,37 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% extends "firefox/australis/fx36/tour-base.html" %}
+{% extends "firefox/base.html" %}
 
 {% set_lang_files "firefox/australis/fx36_tour" %}
```

Comment:
Should this be changed to `firefox/australis/firefox_tour` at this point? I confess that I'm seriously lost at this point, and this looks like a scary change: based on the first file, I'd need to coordinate with the push to production, get rid of fx36_tour.lang and make sure all string are available in firefox_tour.lang